ui/src/components/Common/NumericInput.css

```diff
@@ -0,0 +1,5 @@
+input[type=number]::-webkit-inner-spin-button,
+input[type=number]::-webkit-outer-spin-button {
+  -webkit-appearance: none;
+  margin: 0;
+}
```

Comment:
Can you please change settings in your editor so that it will add \n in every file?  Unix style

Reply:
Fixed